code/testapp/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+}
+
+val mavenCoreVersion: String by project
+
+android {
+      namespace = "com.adobe.campaignclassictestapp"
+
+    defaultConfig {
+        applicationId = "com.adobe.campaignclassictestapp"
+        compileSdk = BuildConstants.Versions.COMPILE_SDK_VERSION
+        minSdk = BuildConstants.Versions.MIN_SDK_VERSION
+        targetSdk = BuildConstants.Versions.TARGET_SDK_VERSION
+        versionCode = BuildConstants.Versions.VERSION_CODE
+        versionName = BuildConstants.Versions.VERSION_NAME
+    }
```

Comment:
Do you want to bring these over from build gradle?
 multiDexEnabled true
         //This says that my test app only wants to use the Phone variant
         missingDimensionStrategy 'target', 'phone'

Reply:
The test app builds fine with the current configuration.. Will add them if required